source/plugins/userprefrences.ts

```diff
@@ -0,0 +1,31 @@
+import debug from "../lib/debug";
+import {
+	Requirements,
+	requires,
+	type IPlugin,
+	type PluginInput,
+} from "../lib/pluginTypes";
+
+class UserprefrencePlugin implements IPlugin {
+	name = "Darkmode";
```

Comment:
```suggestion
class UserpreferencePlugin implements IPlugin {
	name = "User preference";
```
pls change file names too